scripts/auto-rebase/rebase-lvms.sh

```diff
@@ -373,7 +397,7 @@ usage() {
     echo "$(basename "$0") to LVMS_RELEASE_IMAGE         Performs all the steps to rebase LVMS"
     echo "$(basename "$0") download LVMS_RELEASE_IMAGE   Downloads the content of a LVMS release image to disk in preparation for rebasing"
     echo "$(basename "$0") images                        Updates LVMS images"
-    echo "$(basename "$0") manifests                     Updates LVMS manifests"
+    echo "$(basename "$0") manifests LVMS_RELEASE_IMAGE  Updates LVMS manifests"
```

Comment:
I think in the other rebase script the manifests command relies on the image already being downloaded. Instead of changing the interface here, could we have the 'download' command put the information needed by the 'manifests' command somewhere to be found later when 'manifests' is run?